ocs_ci/ocs/elasticsearch.py

```diff
@@ -224,11 +283,13 @@ def get_indices(self):
 
     def _copy(self, es):
```

Comment:
If this function is not used anywhere, it is a good idea to remove it altogether.

Reply:
currently exiting tests (fio / smallfiles) using it, so, in fallow-up PR's i will replace it in the tests and when there will be no used of it i will delete it.